src/test/java/bio/terra/service/filedata/azure/tables/TableServiceClientUtilsTest.java

```diff
@@ -0,0 +1,115 @@
+import com.azure.core.credential.AzureNamedKeyCredential;
+import com.azure.data.tables.TableClient;
+import com.azure.data.tables.TableServiceClient;
+import com.azure.data.tables.TableServiceClientBuilder;
+import com.azure.data.tables.models.TableEntity;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
+@Category(Connected.class)
+public class TableServiceClientUtilsTest {
+  private static final Logger logger = LoggerFactory.getLogger(TableServiceClientUtilsTest.class);
+  private TableServiceClient tableServiceClient;
+  private String tableName;
+
+  @Autowired ConnectedOperations connectedOperations;
+  @Autowired private ConnectedTestConfiguration testConfig;
+  @MockBean private IamProviderInterface samService;
+  @Autowired AzureUtils azureUtils;
+
+  @Before
+  public void setup() throws Exception {
+    connectedOperations.stubOutSamCalls(samService);
+    tableServiceClient =
+        new TableServiceClientBuilder()
+            .credential(
+                new AzureNamedKeyCredential(
+                    testConfig.getSourceStorageAccountName(),
+                    azureUtils.getSourceStorageAccountPrimarySharedKey()))
+            .endpoint(
+                "https://" + testConfig.getSourceStorageAccountName() + ".table.core.windows.net")
+            .buildClient();
+  }
+
+  @After
+  public void cleanup() throws Exception {
+    if (tableName != null) {
+      try {
+        TableClient tableClient = tableServiceClient.getTableClient(tableName);
+        tableClient.deleteTable();
+      } catch (Exception ex) {
+        logger.error("Unable to delete table {}", tableName, ex);
+      }
+    }
+
+    connectedOperations.teardown();
+  }
+
+  @Test
+  public void testUtils() {
+    tableName = Names.randomizeName("testTable123").replaceAll("_", "");
+    TableClient tableClient = tableServiceClient.getTableClient(tableName);
+
+    boolean tableExists = TableServiceClientUtils.tableExists(tableServiceClient, tableName);
+    assertThat("table should not exist", tableExists, equalTo(false));
+
+    tableServiceClient.createTableIfNotExists(tableName);
+    boolean tableExistsNow = TableServiceClientUtils.tableExists(tableServiceClient, tableName);
+    assertThat("table should exist", tableExistsNow, equalTo(true));
```

Comment:
If you're testing for truth, you can use `assertThat(String, boolean)` and skip the matcher.
```suggestion
    assertThat("table should exist", tableExistsNow);
```